### PR TITLE
fixes #1018 allows banners to not contain links

### LINKED ISSUE #1018: Decorative banners
There are use cases for banners that don't link to a page, e.g. as visual separators.

To make this intent explicit, maybe add a checkbox to be ticked for a banner to render without a link,
so that if the referenced page is invalid/blank (e.g. due to some page deletion) it will not accidentally render.

### PR DESCRIPTION
**Pull request by ChrisMarsh82**
updated HTML so link would not be added if it didn't exist.
Added logic to add banners if linked page is published or if there is no link

### CODE REVIEW

**Comment by ChrisMarsh82** [comment]
@ehmadzubair @sheralim012 - can one of you check this is the right way to do the HTML?

File: home/templates/home/tags/banners_list.html [hunk 1]
```diff
@@ -2,8 +2,12 @@
 
 <section class='banner-holder'>
     {% for banner in banners %}
-        <a href="{% pageurl banner.banner_link_page %}">
-            {% image banner.banner_image width-360 %}
-        </a>
+        {% if banner.banner_link_page == None %}
```
Comment (sheralim012) [inline home/templates/home/tags/banners_list.html:5]: `{% if banner.banner_link_page %}` is sufficient

File: home/templates/home/tags/banners_list.html [hunk 2]
```diff
@@ -2,8 +2,12 @@
...
+            <a>{% image banner.banner_image width-360 %}</a>
```
Comment (sheralim012) [inline home/templates/home/tags/banners_list.html:6]: could you try using span instead of a tag as it shouldn't be clickable without linked page

**Review by sheralim012 — CHANGES_REQUESTED** [review]

**Comment by geoo89** [comment]
Obsolete due to #1239

**Pull request closed without merging**